packages/ui-components/src/components/toaster/toaster.tsx

```diff
@@ -111,6 +111,7 @@ export class KvToaster implements IToaster, IToasterEvents {
 						<span class="main-message">{this.header}</span>
 						{!isEmpty(this.description) && <span class="secondary-message">{this.description}</span>}
 					</div>
+					<slot></slot>
 					{this.closable && (
 						<div class="toaster-close-icon">
```

Comment:
Just to prevent future features, I think this slot should be a named slot "right" and we should also have a named left slot too.

Reply:
@djcouto where would you add the left slot?

Reply:
Before the "message-content" and after the icon.

Reply:
@djcouto this would introduce some breaking changes to the slot component. Should i proceed with the change anyway?

Reply:
Not sure which breaking changes you are referring to, but let's keep the component without the left slot and implement it in the future when we need it.